cowin/core/tasks.py

```diff
@@ -44,6 +44,25 @@ def check_slots(*args, **kwargs):
         check_slots_by_district.delay(district_id, dates)
     return
 
+@shared_task(name='core.tasks.repopulate_cache')
+def repopulate_cache(*args, **kwargs):
+    logger.info("running repopulate_cache task")
+    try:
+        users= User.objects.filter(verified=True, is_deleted=False)
```

Comment:
Change this to User.objects.filter(verified=True, is_deleted=False).values('id', 'district_ids')
Other wise a query will run in every loop.